frontend/src/components/Register.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useEffect } from 'react'
+import { useSelector, useDispatch, batch} from 'react-redux'
+import { useHistory, Link } from 'react-router-dom'
+
+import user from '../reducers/user'
+
+import { API_URL } from '../reuseables/urls'
+import { MainContainer, Header, Form, Label, InputField, Text, SecondaryButtonContainer, PrimaryButton, SecondaryButton, ErrorMessage, InputWrapper } from './styled-components/form-style'
+
+const Register = () => {
```

Comment:
your `Register` form is too similar to the `Login` one. You should build one form and pass differences as parameters. 🤓